### PR TITLE
multregt and noaqunnc

### PR DESCRIPTION
**Pull request by tskille**
The following test case is not currently not supported in OPM Flow, but this will hopefully be developed  in not too long time.

### CODE REVIEW

**Comment by tskille** [comment]
This PR has now been updated with one extra test case. 

* AQUNUM-03.DATA  - MULTREGT - Item 5 = ALL
* AQUNUM-04.DATA  - MULTREGT - Item 5 = NOAQUNNC

When running AQNUM-04.DATA the simulator stops with the following error message
```
Error: 
An error occurred while creating the reservoir properties
Internal error: Sorry - currently we do not support 'NOAQUNNC' for MULTREGT.
```

When running AQNUM-03.DATA with flow the simulation completes, but the results are not correct.  In this case Item 5 in MULTREGT is set to ALL which means that all NNC (including NNC generated from numerical aquifer connections) should be multiplied with multiplier in item 3.

When running AQNUM-03.DATA we are getting the behavior we should have gotten from AQUNUM-04.DATA

File: aquifers/AQUNUM-03.DATA [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+RUNSPEC 
+-- ========================================================================
+
+
+TITLE
+   NUM AQUIFER AND MULTREGT
+
+ 
+-- Turns off echoing of all the input files to the print file.  
+NOECHO
```
Comment (bska) [inline aquifers/AQUNUM-03.DATA:35]: We don't really support this keyword so maybe it's better to remove it?

File: aquifers/AQUNUM-03.DATA [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+
+-- Rel perm data - SWOF, SGOF
+-- SWOF: Water-oil saturation tables
+-- SGOF: Gas-oil saturation tables versus gas
+INCLUDE
+  'include/sattab_basemod1.sattab' /
+
+-- Activates stone's first three phase oil relative permeability model. 
+-- There is no data required for this keyword. 
+STONE1
```
Comment (bska) [inline aquifers/AQUNUM-03.DATA:197]: Cool!  I don't think we have a lot of simulation cases which use Stone's first model.

**Review by bska — COMMENTED** [review]
Apart from a couple of instances of `NOECHO` this looks good to me.

**Comment by atgeirr** [comment]
I think this should be merged, so we can get to work on the MULTREGT(5) == ALL bug. But I agree with @bska that the NOECHO should be removed, no need to bury ourselves under warnings if we don't have to...

@tskille will you update this?

**Comment by tskille** [comment]
Sorry I forgot about this. I totally agree that unsupported keywords should be removed from the data decks in opm-tests.

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  I'll merge into master.

**Pull request merged**